src/Duration.php

```diff
@@ -539,9 +539,8 @@ public function getNanos(): int
     public function getTotalMillis(): int
     {
         $millis = $this->seconds * 1000;
-        $millis += intdiv($this->nanos, 1_000_000);
 
-        return $millis;
+        return $millis + intdiv($this->nanos, 1_000_000);
     }
```

Comment:
I'm not fond of this one (and any of those in this commit). I'd either return the whole operation without any variable (on several lines for more readability), or leave it this way.

The way this rule if currently applied, I feel like we're doing one half of a job!

Reply:
I thought this one commit might clash with the current "code style" of the library. I will revert it and skip this rule in `rector.php`.